cmd/stats.go

```diff
@@ -141,15 +141,16 @@ func (ps *PerformanceStats) GetStats() (int64, int64, int64, float64, int64, int
 // Note: This may return slightly stale data since we're not using locks,
 // but this is acceptable for monitoring purposes and eliminates contention
 func (ps *PerformanceStats) GetCurrentSecondStats() (int64, int64, int64, int64, int64) {
-	if ps.currentHistogram == nil || ps.currentHistogram.TotalCount() == 0 {
+	histToUse := ps.secondHistograms[ps.currentSecond-1]
```

Comment:
Might wanna update the comment above about stale data to explain why we are doing this